src/mpyl/schema/mpyl_config.schema.yml

```diff
@@ -360,9 +360,6 @@ definitions:
     type: object
     properties:
       accessKeyId:
-        type: string
+        type: [ string, null ]
       secretAccessKey:
-        type: string
-    required:
-      - accessKeyId
-      - secretAccessKey
+        type: [ string, null ]
```

Comment:
Are these properties independently optional? If they are dependent, you could specify that in the schema like:
```yaml
    dependencies:
      "accessKeyId": [ secretAccessKey ]
      "secretAccessKey": [ accessKeyId ]
```

Reply:
yep, they are dependent - I've updated it